anchore_engine/services/policy_engine/engine/vulns/providers.py

```diff
@@ -1130,8 +1133,20 @@ def _try_load_report_from_store(
 
             return new_report
 
-    def get_vulnerabilities(self, **kwargs):
-        raise NotImplemented
+    def get_vulnerabilities(
+        self, ids, affected_package, affected_package_version, namespace, session
+    ):
+        unmapped_results = GrypeWrapperSingleton.get_instance().query_vulnerabilities(
```

Comment:
This belongs in the GrypeScanner. That allows us to override the scanner behaviour downstream if necessary.

Reply:
I've made that change.